WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/BloggingPromptsAnalyticsTracker.kt

```diff
@@ -0,0 +1,28 @@
+package org.wordpress.android.ui.bloggingprompts
+
+import org.wordpress.android.analytics.AnalyticsTracker.Stat.BLOGGING_PROMPTS_LIST_ACCESSED
+import org.wordpress.android.analytics.AnalyticsTracker.Stat.BLOGGING_PROMPTS_LIST_TAB_CHANGED
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+import java.util.Locale
+import javax.inject.Inject
+
+class BloggingPromptsAnalyticsTracker @Inject constructor(
+    private val analyticsTracker: AnalyticsTrackerWrapper
+) {
+    fun trackScreenAccessed(site: SiteModel, currentTab: PromptSection) =
+            analyticsTracker.track(
+                    BLOGGING_PROMPTS_LIST_ACCESSED,
+                    site,
```

Comment:
I believe prompts are fetched for a _site_ (and there are prompts only if a _site_ has a min number of posts). If the successful loading of the prompts page depends on the _site_, then it made sense to me to track it as a relevant property. This could be useful during debugging purposes too. 

If you feel this property is still unnecessary, I'll remove it. Could you confirm?

Reply:
I believe prompts are fetched for a _site_ (and there are prompts only if a _site_ has a min number of posts). If the successful loading of the prompts page depends on the _site_, then it made sense to me to track it as a relevant property. This could be useful during debugging purposes too. 

If you feel this property is still unnecessary, I'll remove it. Could you confirm?

Reply:
I believe prompts are fetched for a _site_ (and there are prompts only if a _site_ has a min number of posts). If the successful loading of the prompts page depends on the _site_, then it made sense to me to track it as a relevant property. This could be useful during debugging purposes too. 

If you feel this property is still unnecessary, I'll remove it. Could you confirm?

Reply:
> This could be useful during debugging purposes too.

I agree, good idea. Let's leave it, then 👍